### ISSUE TITLE
rucio upload --no-register --pfn fails

### ISSUE THREAD

**Issue by bari12** [rucio/rucio#1377]
Motivation
----------
rucio upload --no-register --pfns fails while it works with 1.16.4.

```
[wguan@lxplus067 wguan]$ /usr/bin/env rucio -v upload --rse CERN-PROD_ES --scope transient --no-register --pfn s3://cs3.cern.ch:443//atlas-eventservice/05cc36b8-383f-444f-a384-592fad3b918e /bin/hostname
2018-07-25 14:54:14,552 WARNING Upload with given pfn implies that no_register is True
2018-07-25 14:54:14,658 INFO    Preparing upload for file hostname
2018-07-25 14:54:16,608 ERROR   An unknown exception occurred.
Details: no error information passed (http status code: 500 ('internal_server_error', 'server_error', '/o\\', '\xe2\x9c\x97'))
Completed in 2.1462 sec.
```

The problem is related to rsemanager.exists call, which trys to resolve the did on the server with lfns2pfns, which does not work as the replica does not exists. In 1.16.4 with bin/rucio it was first checked if the did exists, and if not, rsemanager.exists was not even called.

**Comment by bari12** [comment]
@TWAtGH isn't this fixed with 1.17.6?

**Comment by TWAtGH** [comment]
No. #1395 will fix this.
